### PR TITLE
Add replica schedule client

### PR DESCRIPTION
**Pull request by gabriel-samfira**
This adds the replica schedule client.

### CODE REVIEW

File: coriolisclient/cli/replica_schedules.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        return data
+
+
+class CreateReplicaSchedule(show.ShowOne):
+    """Start a replica schedule"""
+    def get_parser(self, prog_name):
+        parser = super(CreateReplicaSchedule, self).get_parser(prog_name)
+        parser.add_argument('--replica',
+                            help='The ID of the replica')
+        parser.add_argument('--schedule',
```
Comment (alexpilotti) [inline coriolisclient/cli/replica_schedules.py]: Should we pass a cron string to make it easier like "* 1 * * *" for the users?

Comment (gabriel-samfira) [inline coriolisclient/cli/replica_schedules.py]: I was thinking of adding a cron string parser :)

File: coriolisclient/cli/replica_schedules.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                            help='JSON with the schedule')
+        parser.add_argument('--expires_at',
+                            help='ISO8601 formatted date. This is optional',
```
Comment (alexpilotti) [inline coriolisclient/cli/replica_schedules.py]: No need to put "This is optional" in the help, the parser takes care of displaying arguments accordingly.

File: coriolisclient/cli/replica_schedules.py (hunk 1 above)
Comment (alexpilotti) [inline coriolisclient/cli/replica_schedules.py]: Should we pass a cron string to make it easier like "* 1 * * *" for the users?
Here a good help string could... help

File: coriolisclient/v1/replica_schedules.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    def get(self, replica, schedule):
+        return self._get(
+            '/replicas/%(replica_id)s/schedules/%(schedule_id)s' %
+            {"replica_id": base.getid(replica),
+             "schedule_id": base.getid(schedule)},
+            'schedule')
+
+    def _validate_schedule(self, schedule):
+        json.loads(schedule)
```
Comment (alexpilotti) [inline coriolisclient/v1/replica_schedules.py]: "validate" :-)

**Review by alexpilotti — APPROVED** [review]

**Pull request merged**